x-pack/plugins/observability_solution/ux/public/components/app/rum_dashboard/charts/visitor_breakdown_chart.tsx

```diff
@@ -97,7 +103,7 @@ export function VisitorBreakdownChart({
   );
 }
 
-const visConfig: PieVisualizationState = {
+const visConfig = {
```

Comment:
I removed it because we would have to replace it with the PersistedPieVisualizationState (as they just got misaligned) instead and I am not sure if it's worth exposing it to consume it here. I just don't see that it improves our confidence here anyway.